system/nxcodec/Kconfig

```diff
@@ -0,0 +1,30 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config SYSTEM_NXCODEC
+	bool "NxCodec test application"
+	default n
+	---help---
+		Enable support for the NxCodec test application and optional
+		command line interface.
```

Comment:
Please improve this HELP: what is NxCodec? What problem does it solve? How to use it?

Please include this new command here: https://nuttx.apache.org/docs/latest/applications/index.html

Reply:
@shizhenghui please add an introduction to https://nuttx.apache.org/docs/latest/applications/index.html

Reply:
> Please improve this HELP: what is NxCodec? What problem does it solve? How to use it?
> 
> Please include this new command here: https://nuttx.apache.org/docs/latest/applications/index.html

@acassis : I have submitted an `.rst` file for nxcodec, please help to review it.